module/motion/HeadController/src/HeadController.hpp

```diff
@@ -33,7 +33,7 @@ namespace module::motion {
     class HeadController : public NUClear::Reactor {
     private:
         const size_t id;
-        double min_yaw, max_yaw, min_pitch, max_pitch, head_motor_gain, head_motor_torque, p_gain;
+        double min_yaw, max_yaw, min_pitch, max_pitch, head_motor_gain, head_motor_torque, smoothing_factor;
```

Comment:
These values are floats everywhere else, what's the reason for them being doubles here?

Reply:
No reason, will update.